extras/indexing/src/main/java/mvm/rya/indexing/accumulo/geo/GeoMesaGeoIndexer.java

```diff
@@ -435,8 +435,11 @@ public void close() throws IOException {
 
     @Override
     public String getTableName() {
-       return ConfigUtils.getGeoTablename(conf);
+            return getTableName(conf);
     }
+    public static String getTableName(Configuration conf) {
```

Comment:
I'd like to make it not use the conf object at all. When we switch to the RyaDetails, it has to be changed anyway.  Also the table prefix is an implementation detail.